### PR TITLE
new command that displays quay mirrors

### PR DESCRIPTION
**Pull request by jmelis**
Example:

```
$ qontract-cli --config config.prod.toml get mirrors
REPO                                               UPSTREAM                                     PUBLIC
-------------------------------------------------  -------------------------------------------  --------
quay.io/REDACTED/REDACTED                          REDACTED                                     True
```

### CODE REVIEW

File: reconcile/quay_mirror.py [hunk 1]
```diff
@@ -81,11 +56,11 @@ def run(self):
                     _LOG.error('[%s]', details)
 
     def process_repos_query(self):
-        result = self.gqlapi.query(self.QUAY_REPOS_QUERY)
+        apps = queries.get_quay_repos()
```
Comment (apahim) [inline reconcile/quay_mirror.py:59]: grat, tks for that :)

File: tools/qontract_cli.py [hunk 2]
```diff
@@ -349,6 +349,40 @@ def integrations(ctx):
     print_output(ctx.obj['output'], environments, columns)
 
 
+@get.command()
+@click.pass_context
+def mirrors(ctx):
```
Comment (apahim) [inline tools/qontract_cli.py:354]: there are other mirrors... should we be specific here? I'd expect the command to be `get quay-mirrors`

**Review by apahim — APPROVED** [review]
lgtm, just a naming question.

File: tools/qontract_cli.py (hunk 2 above)
Comment (rporres) [reply tools/qontract_cli.py:354]: +1

**Review by rporres — COMMENTED** [review]
LGTM. Just the naming thing...

**Review by apahim — APPROVED** [review]
lgtm

**Pull request merged**